lib/elixir/lib/io.ex

```diff
@@ -141,7 +141,7 @@ defmodule IO do
       non-textual inputs.
 
     * if `line_or_chars` is `:line`, the device is iterated line by line.
-      CRFL newlines  ("\r\n") are automatically normalized to "\n".
+      CRLF newlines  ("\\r\\n") are automatically normalized to "\\n".
```

Comment:
This is often achieved by using the [`~S` sigil](https://hexdocs.pm/elixir/main/Kernel.html#sigil_S/2) rather than manually escaping (you can see we have a lot of `@doc ~S` in the codebase).